include/sdf/Filesystem.hh

```diff
@@ -128,6 +128,14 @@ namespace sdf
       IGN_UTILS_UNIQUE_IMPL_PTR(dataPtr)
     };
   }
+
+  /// \brief The source path replacement if it was parsed from a string,
+  /// instead of a file.
+  const std::string sdfStringSource = "<data-string>";
+
+  /// \brief The source path replacement if the urdf was parsed from a string,
+  /// instead of a file.
+  const std::string urdfStringSource = "<urdf-string>";
```

Comment:
nit This and above violate GSG for constant names:
https://google.github.io/styleguide/cppguide.html#Constant_Names

@azeey Is this what `libsdformat` normally does (not using `kCameCase`), or should it use GSG recommendations?

Reply:
Found some references in `ign-sensors`, https://github.com/ignitionrobotics/ign-sensors/blob/ign-sensors5/src/AirPressureSensor.cc#L42-L51.

I'll wait and see what @azeey thinks

Reply:
I somehow missed these comments, but yes, we should use `kCamelCase`.